draft-kwbdgrr-tsvwg-net-collab-rqmts.md

```diff
@@ -99,35 +99,24 @@ informative:
-When packet contents are encrypted, the approach of using implicit signals is no longer viable.
-
-Bandwidth constraints exist most predominantly at the access network (e.g., radio access networks).
-Users who are serviced via these networks use hosts which run various application clients; each having different connectivity needs for an optimal user experience.
-These needs are not frozen but change over time depending on the application and even depending on how an application is used (e.g., user's preferences).
-An explicit signal to the host can help to manage the use of available bandwidth better and better share it with requesting applications.
+Wireless networks including 5G and WLAN inherently experience large variations in link quality over sub-RTT intervals and on the other hand applications such as interactive media demand both low latency and high bandwidth. Maximizing network utilization and end user experience under such conditions is challenging. Factors that affect wireless networks include change in channel conditions, interference between proximate cells and end user movement. These variations in link quality can be in the order of a millisecond or less {{5G-Lumos}} while congestion control takes several tens of milliseconds (more than one round-trip time (RTT)) to estimate data rate. Similarly, application servers that encode and serve live or interactive media take time to adjust the encoding level and other processes to match the network rate. End-to-end congestion control algorithms are far from being optimal when the link quality is highly variable in sub-RTT timeframes and the application demands both low latency and high bandwidth (e.g., Section 2.1 of {{?RFC6077}}). In these conditions, applications settle for a lower throughput when latency is prioritized, or for higher throughput at the expense of much higher delays.
+
+While rate control based on feedback for a flow (UDP 4-tuple) is evidently not able adapt for sub-RTT changes in available wireless channel resources, the application server can provide information on a per-packet basis that a network shaper may use to utilize the available resources more effectively. {{5G-Octopus}} has shown for volumetric video packets and a rate controller that errs on the side of overestimation, that the network shaper can drop low priority frames of a group of pictures (corresponding to transient wireless link bandwidth drops) and still achieve significantly better performance than state-of-the-art based on feedback. With not fully encrypted packets, networks may use heuristics to build an "implicit signal" gleaned from a packet to prioritize or otherwise shape flows. However, implicit signals are not desirable as they lead to ossification of protocols as result of introducing unintended dependencies {{?RFC9419}}. When packet contents are encrypted, the approach of using implicit signals is no longer viable.
```

Comment:
```suggestion
While rate control based on feedback for a flow (UDP 4-tuple) is evidently not able adapt for sub-RTT changes in available wireless channel resources, an application server can provide information on a per-packet basis that a network shaper may use to allocate the available resources more effectively. {{5G-Octopus}} has shown for volumetric video packets and a rate controller that errs on the side of overestimation, that the network shaper can drop low priority frames of a group of pictures (corresponding to transient wireless link bandwidth drops) and still achieve significantly better performance than state-of-the-art based on feedback. With not fully encrypted packets, networks may use heuristics to build an "implicit signal" gleaned from a packet to prioritize or otherwise shape flows. However, implicit signals are not desirable as they lead to ossification of protocols as result of introducing unintended dependencies {{?RFC9419}}. When packet contents are encrypted, the approach of using implicit signals is no longer viable.
```